### PR TITLE
Make tests pass without Docker

### PR DESCRIPTION
**Pull request by reidpr**

### CODE REVIEW

File: test/chtest/Build [hunk 1]
```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+# Build an Alpine Linux image roughly following the chroot(2) instructions:
+# https://wiki.alpinelinux.org/wiki/Installing_Alpine_Linux_in_a_chroot
+#
+# We deliberately do not sudo. It's a little rough around the edges, because
+# apk expects root, but it better follows the principle of least principle. We
```
Comment (trandles-lanl) [inline test/chtest/Build]: "principle of least privilege"

**Comment by reidpr** [comment]
Read-only on the default binds is because there's no need to change them, so declare that fact explicitly.

User's home directory isn't one of the default binds. It's handled separately.

**Comment by trandles-lanl** [comment]
Overall, looks sane to me.  Are there any remaining known-to-fail tests?

**Comment by reidpr** [comment]
No. I try to avoid known-to-fail tests, and if any do crop up, they will `skip`.

I'll make the requested changes, thanks.

**Pull request merged**